docs/tbtadm.t2t

```diff
@@ -71,6 +73,12 @@ Print the ACL content in the following format:
 UUID    Vendor    Device name    Currently connected?
 ```
 
+: **add** <route-string>
+Add a device to ACL. The argument selects the device to be added by its
```

Comment:
btw can we also select device to be added by uuid?

Reply:
To have the device details to save (vendor and device names), the device must be connected, so route-string makes more sense. We may want to add only UUIDs if we start syncing FW ACL with SW ACL but for this we first need the kernel-side support.